e2e/app/page/admin-workspace-page.ts

```diff
@@ -43,21 +44,30 @@ export default class WorkspaceAdminPage extends AuthenticatedPage {
     await waitWhileLoading(this.page);
   }
 
-  // get the h2 page header (workspace)
-  async getWorkspaceHeader(): Promise<string> {
-    const h2 = await this.page.waitForXPath('//h2', { visible: true });
-    return getPropValue<string>(h2, 'textContent');
+  getLockWorkspaceButton(): Button {
+    return Button.findByName(this.page, { normalizeSpace: LinkText.LockWorkspace });
+  }
+
+  async clickLockWorkspaceButton(): Promise<LockWorkspaceModal> {
+    const button = this.getLockWorkspaceButton();
+    await button.click();
+    const modal = new LockWorkspaceModal(this.page);
+    await modal.waitForLoad();
+    return modal;
   }
 
-  // get all the h3 headers
-  async getAllHeadings3(): Promise<string[]> {
```

Comment:
Deleted these function since they were only verifying the Headings of the workspace-admin page.